config/karma.saucelabs.js

```diff
@@ -119,19 +114,50 @@ module.exports = function(config) {
 
     customLaunchers: sauceLabsBrowsers,
 
-    reporters: ['spec', 'saucelabs'],
+    reporters: ['spec', 'summary', 'saucelabs'],
 
     port: 9876,
 
-    retryLimit: 3,
+    retryLimit: 0,
 
     // concurrency: 10,
 
+    client: {
+      mocha: {
+        timeout: 20000,
```

Comment:
I assume these values are copied from the main karma.js or similar?  If we can't share code, perhaps add a comment referencing where these were copied from so we know how these values were chosen and can try to keep things in sync over time.

Reply:
Created a mocha base config file which all karma configs would read from. This new file will be the single source of truth.